src/store/reducers/user.js

```diff
@@ -1,17 +1,33 @@
 /* eslint-disable linebreak-style */
-import { SET_USER } from '../actions/actionTypes';
+import { SET_USER, FETCH_USER_DATA_REQUEST, FETCH_USER_DATA_SUCCESS, FETCH_USER_DATA_FAILURE, SIGN_USER_OUT} from '../actions/actionTypes';
 
 const initialState = {
-  user: {}
+  loading:'false',
```

Comment:
Codacy found an issue: [Missing space before value for key 'loading'.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5714105)